gtclang/test/atlas-integration-test/AtlasIntegrationTestCompareOutput.cpp

```diff
@@ -0,0 +1,196 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
```

Comment:
there is generally no documentation on what's going on in this file - can we have some broad categories? Most likely not everyone will have a good atlas syntax understanding

Reply:
Yes, documentation would be nice. What do you mean by broad categories? Could you comment with an example?